# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/07
# allow using nanoflann 1.5.0, which provides nanoflann::SearchParameters(), but not nanoflann::SearchParams()
diff -ru dgl-1.1.1/src/graph/transform/cpu/kdtree_ndarray_adapter.h dgl-1.1.1_nanoflanSearchParams/src/graph/transform/cpu/kdtree_ndarray_adapter.h
--- dgl-1.1.1/src/graph/transform/cpu/kdtree_ndarray_adapter.h	2023-06-23 07:00:04.000000000 +0200
+++ dgl-1.1.1_nanoflanSearchParams/src/graph/transform/cpu/kdtree_ndarray_adapter.h	2023-07-06 12:17:16.433558692 +0200
@@ -74,7 +74,7 @@
       FloatType* out_dists) const {
     nanoflann::KNNResultSet<FloatType, IdType> resultSet(num_closest);
     resultSet.init(out_idxs, out_dists);
-    index_->findNeighbors(resultSet, query_pt, nanoflann::SearchParams());
+    index_->findNeighbors(resultSet, query_pt, nanoflann::SearchParameters());
   }

   /** @brief Interface expected by KDTreeSingleIndexAdaptor */
